code/modules/projectiles/effects.dm

```diff
@@ -160,4 +160,16 @@
 	icon_state = "impact_plasma_pink"
 
 /obj/effect/projectile/plasma/impact/heavy
-	icon_state = "impact_plasma_blue"
\ No newline at end of file
+	icon_state = "impact_plasma_blue"
+
+//----------------------------
+// Cutter
+//----------------------------
+/obj/effect/projectile/laser/plasmacutter/tracer
+	icon_state = "plasmacutter"
+
+/obj/effect/projectile/laser/plasmacutter/impact
+	icon_state = "impact_plasmacutter"
+
+/obj/effect/projectile/laser/plasmacutter/muzzle
+	icon_state = "muzzle_plasmacutter"
```

Comment:
Add an empty line at the end of the file.

Reply:
Done